infra/examples-dev/gcp-google-workspace/main.tf

```diff
@@ -22,6 +22,10 @@ terraform {
   #  }
 }
 
+provider "google" {
+  impersonate_service_account = var.gcp_terraform_sa_account_email
```

Comment:
Is this working with a variable? If so cool, because other times it was doing nothing here

Reply:
by default, it's `null` and does nothing; if you pass the variable, it does the impersonation. (AWS case is similar). So yeah useful to have this flexibility in the example. While not a lot of people use this type of approach in practice, imho it's better from security perspective (as users not themselves privileged; just SAs are which can be assumed)